src/agent/block_store_services/block_store_fs.js

```diff
@@ -38,6 +38,10 @@ class BlockStoreFs extends BlockStoreBase {
         // stores the cached df data for the root path
         this.cached_df_data = null;
 
+        // If this is set to true then metadata files
+        // are used instead of xattrs
+        this.xattr_fallback = false;
```

Comment:
I prefer this to be called xattr_enabled because this might be changed later to come from a configuration of the backing store for example, instead of only detected on runtime, and it makes more sense to me as a "positive" condition.